js/src/personal/riyuan/components/about/About.module.css

```diff
@@ -0,0 +1,19 @@
+.about{
+  padding: 88px;
+  min-height: 100vh;
+  position: relative;
+  box-sizing: border-box;
+  background: light-dark(var(--mantine-color-black), var(--mantine-color-white));
+}
+
+.title{
+  flex: 0 0 100%;
+  max-width: 100%;
```

Comment:
nit: Does `max-width: 100%;` do anything?

Reply:
to keep the size of title not overflow